src/cuga/backend/slash_commands/dispatcher.py

```
@@ -0,0 +1,316 @@
+        slash_registry=slash_registry,
+        skill_registry=skill_registry,
+        thread_id=thread_id,
+        clear_stop_event=clear_stop_event,
+        extra=extra or {},
+    )
+
+    builtin = slash_registry.get_builtin(parsed.name)
+    if builtin is not None:
+        try:
+            result = await builtin.handle(ctx)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)